DevEdu.API/DevEdu.DAL/Repositories/CourseRepository.cs

```diff
@@ -166,5 +168,29 @@ public List<CourseTopicDto> SelectAllTopicsByCourseId(int courseId)
                 )
                 .ToList();
         }
+        public void UpdateCourseTopicsByCourseId(int courseId, List<CourseTopicDto> topics)
+        {
+            foreach (var topic in topics)
+            {
+                _connection.Query(
+                    _updateCourseTopicsProcedure,
```

Comment:
we might have ~50 topics in a course
and it means that we'll send 50 requests to DB here
I think it's not the best solution(